addons/web_editor/controllers/main.py

```diff
@@ -310,14 +310,29 @@ def get_assets_editor_resources(self, key, get_views=True, get_scss=True, bundle
                         if excluded_url_matcher.match(url):
                             continue
 
+                        # Set files URL filters
+                        custom_css_files = [
+                            '/website/static/src/scss/bootstrap_variables_override.scss',
+                            '/website/static/src/scss/css_custom_rules.scss',
+                            '/website/static/src/scss/css_custom_variables.scss']
```

Comment:
See my comment about the file names. You should just check it starts by 'user_custom_' then.